neo/Network/P2P/Payloads/ConsensusPayload.cs

```diff
@@ -119,6 +124,7 @@ void IVerifiable.SerializeUnsigned(BinaryWriter writer)
             writer.Write(PrevHash);
             writer.Write(BlockIndex);
             writer.Write(ValidatorIndex);
+            writer.Write(0);
```

Comment:
For the old version payload, it needs to preserve the Timestamp, or else the payload will be broken when forwarding it along by re-serializing.

Reply:
It won't. Because the consensus nodes will be upgraded first.

Reply:
If CN nodes were upgraded first none, of the existing seed nodes would forward any of the consensus messages from them. I don't think thi sis a good idea.

Reply:
Why they don't forward the new consensus messages?

Reply:
Since the old code didn't deserialize the messages from the payload, they would forward them since this now is in the same format. However, is it really a good idea to not allow anyone to upgrade until the CN nodes are upgraded?

Reply:
So with 7b30e10 I think it is acceptable if CN nodes were upgraded first, but I don't think it is ideal to require that this code cannot work with old CN nodes. There are other networks besides the official one run by NEO MainNet; they may not wish to have the same upgrade plan as you do.

Reply:
I changed it, and now we allow non-consensus nodes to be upgraded first.